workspaces/analytics/src/interfaces/Events.ts

```diff
@@ -0,0 +1,26 @@
+//@ts-ignore
+import keymirror from 'keymirror';
+export const Events = keymirror({
+  //on-boarding
+  ApiCreated: null,
+  ApiCheckCompleted: null,
+
+  //Errors
+  JavascriptErrorDetectedInFrontend: null,
+  RequirementForDiffsToHaveASuggestionFailed: null,
```

Comment:
Lou might have a suggestion for assigning these numeric codes or something to make it easier to add more, otherwise seems fine